forums/views.py

```diff
@@ -95,155 +116,170 @@ def account_profile(request, username):
-        context['form'] = ProfileForm(user, instance = instance)
-        return render(request, 'forums/templates/profile.html', context) 
-# view all profile details saved for the user, when clicked on my profile  
+        context['form'] = ProfileForm(user, instance=instance)
+        return render(request, 'forums/templates/profile.html', context)
+
+
+# view all profile details saved for the user, when clicked on my profile
 @login_required
 def account_view_profile(request, user_id):
     # user_id = username
-    user = User.objects.get(pk = user_id)
-    profile = Profile.objects.get(user = user)
+    user = User.objects.get(pk=user_id)
+    profile = Profile.objects.get(user=user)
     flag = False
     # prof = None
     marker = 0
     if 'marker' in request.GET:
         marker = int(request.GET['marker'])
     # if profile:
     #     prof = profile[0]
-    
+
     total = Question.objects.filter(user_id=user_id).count()
     total = int(total - (total % 10 - 10))
-    questions = Question.objects.filter(user_id=user_id).order_by('date_created').reverse()[marker:marker+10]
+    # exceeding the limit so broken
+    questions = (Question.objects.filter(user_id=user_id)
+                 .order_by('date_created').reverse()[marker:marker+10])
     total1 = Answer.objects.filter(uid=user_id).count()
-    total1= int(total1 - (total1 % 10 - 10))
-    answers =Answer.objects.filter(uid=user_id).order_by('date_created').reverse()[marker:marker+10]
+    total1 = int(total1-(total1 % 10 - 10))
+    answers = (Answer.objects.filter(uid=user_id)
+               .order_by('date_created').reverse()[marker:marker+10])
 
     if str(user_id) == str(request.user.id):
-         flag = True
-
+        flag = True
 
     instance = Profile.objects.get(user_id=user.id)
-    form = ProfileForm(user, instance = instance)
+    form = ProfileForm(user, instance=instance)
     context = {
         'show': flag,
-        'profile' : profile,
-        'media_url' : settings.MEDIA_URL,
-        'questions' : questions,
-        'answers' : answers,
-        'form' : form,
-        'user_show':user,
-        
+        'profile': profile,
+        'media_url': settings.MEDIA_URL,
+        'questions': questions,
+        'answers': answers,
+        'form': form,
+        'user_show': user,
     }
     return render(request, 'forums/templates/view-profile.html', context)
-    
-    
-                
-# send confirm registration link    
+
+
+# send confirm registration link
 def send_registration_confirmation(user):
     p = Profile.objects.get(user=user)
-     
+
     # Sending email when an answer is posted
     subject = 'Account Active Notification'
     message = """Dear {0},\n
-    Thank you for registering at {1}.\n\n You may activate your account by clicking on this link or copying and pasting it in your browser
+    Thank you for registering at {1}.\n\n
+    You may activate your account by clicking on this
```

Comment:
Starting on a newline after add \n seems unnecessary. Please rewrite this in a single block of text and no need of starting on a newline for eg. Regards,\nFOSSEE forum\nIIT Bombay is good enough